packages/sw-routing/src/lib/router.js

```diff
@@ -52,11 +52,24 @@ class Router {
   /**
    * Start with an empty array of routes, and set up the fetch handler.
    */
-  constructor() {
+  constructor({handleFetch} = {}) {
+    if (typeof handleFetch === 'undefined') {
+      handleFetch = true;
+    }
+
```

Comment:
Yeah - this is largely just a debug feature (i.e. Web Starter Kit for dev mode skipping fetch to always getting latest from network).